index.js

```diff
@@ -60,7 +64,11 @@ function config(opts) {
             cfg[name] = value;
         }
         else if ( opt.type === 'boolean' ) {
-            booleanValue = valid.boolean[value.toLowerCase()];
+            if ( typeOf(value) != 'boolean' ) {
```

Comment:
Shouldn't this be `typeof`. I think `typeOf` is a typo.

(Also, it's more preferable to be `typeof value` instead of `typeof(value)`.)

It may also be better to be more explicit and check for `typeof value === 'string'` and do the conversion, then an else if `typeof value === 'boolean'` and use it directly, and then we throw on an unknown type like we currently do. How does that sound?